.github/workflows/prerelease.yml

```diff
@@ -0,0 +1,66 @@
+name: pre-release
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  pre-release:
+    if:  contains(github.event.comment.body, 'prerelease')
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x]
+
+    steps:
+      - uses: actions/github-script@v1
```

Comment:
You are using many versions of this action (0.3.0, v1, v4) is there a reason for it? If so, please explain in code in comments.

Reply:
The prerelease/release scripts should be reviewed by @chenele

Reply:
@chenele can you take a look?